crates/pop-contracts/src/utils/contracts_node.rs

```diff
@@ -36,50 +44,78 @@ pub async fn is_chain_alive(url: url::Url) -> Result<bool, Error> {
 	}
 }
 
+/// A supported chain.
+#[derive(Debug, EnumProperty, PartialEq, VariantArray)]
+pub(super) enum SoloChain {
+	/// Parachain containing core Polkadot protocol features.
+	#[strum(props(
+		Repository = "https://github.com/paritytech/substrate-contracts-node",
+		Binary = "substrate-contracts-node",
+		TagFormat = "{tag}",
+		Fallback = "v0.41.0"
+	))]
+	ContractNode,
+}
+
+impl TryInto for SoloChain {
+	/// Attempt the conversion.
+	///
+	/// # Arguments
+	/// * `tag` - If applicable, a tag used to determine a specific release.
+	/// * `latest` - If applicable, some specifier used to determine the latest source.
+	fn try_into(&self, tag: Option<String>, latest: Option<String>) -> Result<Source, Error> {
+		let archive = archive_name_by_target()?;
+		let archive_bin_path = release_folder_by_target()?;
+		Ok(match self {
+			&SoloChain::ContractNode => {
+				// Source from GitHub release asset
+				let repo = GitHub::parse(self.repository())?;
+				Source::GitHub(ReleaseArchive {
+					owner: repo.org,
+					repository: repo.name,
+					tag,
+					tag_format: self.tag_format().map(|t| t.into()),
+					archive,
+					contents: vec![(archive_bin_path, Some(self.binary().to_string()))],
+					latest,
+				})
+			},
+		})
+	}
+}
+
+impl pop_common::sourcing::traits::Source for SoloChain {}
+
 /// Checks if the `substrate-contracts-node` binary exists
-/// or if the binary exists in pop's cache.
 /// returns:
 /// - Some("", <version-output>) if the standalone binary exists
-/// - Some(binary_cache_location, "") if the binary exists in pop's cache
 /// - None if the binary does not exist
-pub fn does_contracts_node_exist(cache: PathBuf) -> Option<(PathBuf, String)> {
-	let cached_location = cache.join(BIN_NAME);
+pub fn standalone_binary_exists() -> Option<(PathBuf, String)> {
```

Comment:
Can probably just return `Option(<Version: String>)` because the standalone binary won't have a path.